src/System.Web.HttpUtility/src/System/Web/Util/HttpEncoder.cs

```diff
@@ -570,7 +570,7 @@ private static byte[] UrlEncodeNonAscii(byte[] bytes, int offset, int count, boo
         }
 
         [Obsolete("This method produces non-standards-compliant output and has interoperability issues. The preferred alternative is UrlEncode(*).")]
-        internal static string UrlEncodeUnicode(string value, bool ignoreAscii)
+        internal static string UrlEncodeUnicode(string value)
```

Comment:
Shouldn't all call sites be changed as part of this commit?

Reply:
Ah, I see it's in the next commit.

Reply:
That was a slip-up if they weren't in the same commit, but the merging will make it all the one, so I won't change it.